### PR TITLE
Export the LDA visualisation to a HTML file

### PR DESCRIPTION
**Pull request by mermop**
Creates a module, `bagel.viz`, that outputs the LDA visualisation to a HTML file.

Not sure how idiomatic to Python the way I've structured this is, but it works..

### CODE REVIEW

File: bagel/viz/viz.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+from bagel import DataSet, DataFile
+import pyLDAvis.gensim
+
+def export_lda_viz(num_topics, file_pattern, output_path):
+    dataset = DataSet(glob=file_pattern, num_topics=num_topics)
+    dataset.make_lda_model()
+
+    lda_display = pyLDAvis.gensim.prepare(dataset.lda_model, dataset.corpus, dataset.dictionary, sort_topics=False)
```
Comment (Br3nda) [inline bagel/viz/viz.py]: :sparkling_heart:

File: export_lda_viz.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from bagel.viz.viz import export_lda_viz
+
+FILE_PATTERN = 'zero-carbon-bill/input/*.txt'
+# Tweak this parameter until it brings joy
+NUMBER_OF_TOPICS = 5
+OUTPUT_PATH = 'docs/index_lda.html'
+
+export_lda_viz(NUMBER_OF_TOPICS, FILE_PATTERN, OUTPUT_PATH)
```
Comment (Br3nda) [inline export_lda_viz.py]: should be wrapped in main

Comment (Br3nda) [inline export_lda_viz.py]:
```suggestion
if __name__ == "__main__":
    export_lda_viz(NUMBER_OF_TOPICS, FILE_PATTERN, OUTPUT_PATH)
```

Comment (mermop) [reply export_lda_viz.py]: Hahahaha, thank you Brenda, I was thinking about sneakily adding you to review <3

Comment (mermop) [reply export_lda_viz.py]: does it look all right to merge @Br3nda ?

**Review by JacOng17 — APPROVED** [review]
LGTM, the html file works :+1:

**Pull request merged**